Lib/linecache.py

```diff
@@ -76,6 +78,12 @@ def checkcache(filename=None):
 
 
 def updatecache(filename, module_globals=None):
+    # These imports are not at top level because linecache is in the critical
+    # path of the interpreter startup and importing os and sys take a lot of time
+    # and slow down the startup sequence.
+    import os
+    import sys
+
     """Update a cache entry and return its list of lines.
```

Comment:
We lost the docstring because you put the imports before it. Filed #117948 to fix this.